### ISSUE TITLE
The main frame is blank without network

### ISSUE THREAD

**Issue by exaos** [senshu/Sozi#395]
The main frame cannot be displayed if there is no network link or just because I am behind the China Great Firewall. I think, the reason might be that I cannot load this `client.js`.
```
./resources/app/index-webapp.html:18:        <script src="https://apis.google.com/js/cliet.js" defer></script>
./resources/app/index.html:18:        <script src="https://apis.google.com/js/client.js" efer></script>
```

**Comment by senshu** [comment]
> I think, the reason might be that I cannot load this client.js.

Have you tried to remove this line from `index.html`?
The Google API is used only in the web-hosted editor (to open files from Google Drive), so you can safely remove it from the desktop application.

**Comment by exaos** [comment]
Thanks! I tried to remove this line, then it worked very well.

**Comment by senshu** [comment]
OK. I have pushed a fix to the master branch (commit 72ac2d67ef2ca96d6106ae62c2e2d96cb9c75299).
